### ISSUE TITLE
try to reproduce nice-looking filters in 2010 nips paper

### ISSUE THREAD

**Issue by zym1010** [leelabcnbc/unsup-pytorch#1]
this will be done under <https://github.com/leelabcnbc/unsup-pytorch/tree/master/script/2010_paper>

given that convPSD really doesn't affect the learning of dictionary, probably it's easier to to use `ConvSC` directly.

- [x] get all data (1000000 images of 25x25)
- [x] set up a basic main loop for training, with some stats spitted out every hundreds of iterations.
- [x] try various parameters (lambda)
- [x] fix the loss computation (should be averaged over images in a single minibatch)
- [x] remember to add `normalize`.
- [x] set up some parameter saving (plotting) mechanisms.

**Comment by zym1010** [comment]
check #2. I can surely can get nice results by running <https://github.com/leelabcnbc/unsup-pytorch/blob/master/script/2010_paper/run_batch_20180614.py>.
